services/api-server/src/simcore_service_api_server/api/routes/solvers_jobs.py

```diff
@@ -100,53 +98,42 @@ async def create_job(
     NOTE: This operation does **not** start the job
     """
 
-    async def _fake_impl():
-        from ._jobs_faker import create_job_impl
-
-        solver = await catalog_client.get_solver(user_id, solver_key, version)
-        return await create_job_impl(solver.id, solver.version, inputs, url_for)
+    # ensures user has access to solver
+    solver = await catalog_client.get_solver(user_id, solver_key, version)
 
-    async def _draft_impl():
+    # creates NEW job as prototype
+    pre_job = Job.create_solver_job(solver=solver, inputs=inputs)
+    logger.debug("Creating Job '%s'", pre_job.name)
 
-        # ensures user has access to solver
-        solver = await catalog_client.get_solver(user_id, solver_key, version)
+    # -> catalog
+    # TODO: validate inputs against solver input schema
 
-        # creates NEW job as prototype
-        pre_job = Job.create_solver_job(solver=solver, inputs=inputs)
-        logger.debug("Creating Job '%s'", pre_job.name)
+    #   -> webserver:  NewProjectIn = Job
+    project_in: NewProjectIn = create_new_project_for_job(solver, pre_job, inputs)
+    new_project: Project = await webserver_api.create_project(project_in)
+    assert new_project  # nosec
+    assert new_project.uuid == pre_job.id  # nosec
```

Comment:
Question: how are errors rendered to the final user of the API at this point?

I'd actually prefer you explicitly raised more meaningful errors or errors which hide the implementation details for the users. Because this API is the one exposed to the users (public API).

Same considerations apply for all asserts below.

Also, I am not sure, will these still work on production environments where debug mode is generally disabled?